helm-chart/eoapi/values.yaml

```diff
@@ -401,6 +463,25 @@ vector:
       PORT: "8080"
       # https://www.uvicorn.org/settings/#production
       WEB_CONCURRENCY: "5"
+    # Additional environment variables
+    extraEnvVars: []
 
 docServer:
   enabled: true
+
+# TODO DEPRECATE THIS SECTION
```

Comment:
I would prefer we create a separate deployment chart containing this. Could even live in the same repo. Perhaps `helm-chart/eoapi-azure`. But it seems we add unnecessary tech debt with with cloud provider specific code.